### PR TITLE
Fixed #26542 -- Fixed quoting of name in Postgres CreateExtension operation

### PR DESCRIPTION
**Pull request by conradev**
https://code.djangoproject.com/ticket/26542

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
Looks like you need to add `CreateExtension = mock.Mock()` for non-PG test runs as done for the other operations.

File: tests/postgres_tests/migrations/0001_setup_extensions.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        CreateExtension('uuid-ossp'),
```
Comment (timgraham) [inline tests/postgres_tests/migrations/0001_setup_extensions.py]: A comment explaining the purpose here wouldn't hurt.

**Comment by conradev** [comment]
I made the suggested changes, let me know if I should change anything else!

**Comment by timgraham** [comment]
merged in c11219833253d42d72f4a48a6c9e3a4944c27391, thanks!

**Pull request closed without merging**